### PR TITLE
feat(build): support android cordova verbose option

### PR DESCRIPTION
**Pull request by daniellacosse**

### CODE REVIEW

File: src/cordova/setup.action.mjs [hunk 1]
```diff
@@ -46,23 +43,30 @@ export async function main(...parameters) {
...
 
   if (buildMode === 'debug') {
-    console.warn(`WARNING: setting up "${cordovaPlatform}" in [DEBUG] mode. Do not publish this build!!`);
+    console.warn(`WARNING: setting up "${outlinePlatform}" in [DEBUG] mode. Do not publish this build!!`);
   }
 
-  await runAction('www/build', outlinePlatform, `--buildMode=${buildMode}`);
-
-  await rmfr(`platforms/${cordovaPlatform}`);
+  await rmfr('www');
```
[the remark this answers is not in the document]
Comment (jyyi1) [reply src/cordova/setup.action.mjs:49]: why not adding them to `npm run clean`?

File: src/cordova/setup.action.mjs [hunk 2]
```diff
@@ -46,23 +43,30 @@ export async function main(...parameters) {
...
+  await rmfr('platforms');
   await rmfr('plugins');
 
-  if (!existsSync(path.resolve(getRootDir(), 'platforms', cordovaPlatform))) {
+  await runAction('www/build', outlinePlatform, `--buildMode=${buildMode}`);
+
+  if (cordovaPlatform === 'osx') {
     await cordova.platform(
       'add',
       [cordovaPlatform === 'osx' ? `github:apache/cordova-osx#${WORKING_CORDOVA_OSX_COMMIT}` : cordovaPlatform],
```
Comment (jyyi1) [inline src/cordova/setup.action.mjs:58]: This check is redundant now

```suggestion
      [`github:apache/cordova-osx#${WORKING_CORDOVA_OSX_COMMIT}`],
```

**Review by jyyi1 — APPROVED** [review]

File: src/cordova/build.action.mjs [hunk 3]
```diff
@@ -70,7 +70,11 @@ export async function main(...parameters) {
         '--gradleArg=-PcdvBuildMultipleApks=true',
       ];
     }
+
+    cordova.on('verbose', console.debug);
+
     await cordova.compile({
+      verbose: buildMode === 'debug',
```
Comment (bemasc) [inline src/cordova/build.action.mjs:77]: I'm not sure this makes sense.  Why would we only want detailed build logs when building the debug version?  I think they're probably independent questions.

Comment (jyyi1) [reply src/cordova/build.action.mjs:77]: Maybe add an additional argument to action? `npm run action cordova/build android -- -verbose`

Comment (fortuna) [reply src/cordova/build.action.mjs:77]: I agree with Ben. Verbose and debug are separate things.

Perhaps we should just forward the extra params from the `npm run action` to the cordova call

File: src/cordova/setup.action.mjs (hunk 1 above)
[the remark this answers is not in the document]
Comment (fortuna) [reply src/cordova/setup.action.mjs:49]: This may make Cordova complain it's not a valid Cordova project.

File: src/cordova/build.action.mjs [hunk 4]
```diff
@@ -27,7 +27,7 @@ import {execSync} from 'child_process';
  * @param {string[]} parameters
  */
 export async function main(...parameters) {
-  const {platform: cordovaPlatform, buildMode} = getCordovaBuildParameters(parameters);
+  const {platform: cordovaPlatform, buildMode, verbose} = getCordovaBuildParameters(parameters);
   const outlinePlatform = cordovaPlatform === 'osx' ? 'macos' : cordovaPlatform;
 
   await runAction('cordova/setup', ...parameters);
```
Comment (fortuna) [inline src/cordova/build.action.mjs:33]: FYI, if you'd like to be thorough, you can pass `-verbose` to the `xcodebuild` calls too.

**Review by fortuna — APPROVED** [review]

**Review by bemasc — APPROVED** [review]
I think this should also update the README to document the new flag.

File: .github/workflows/build_and_test_debug.yml [hunk 5]
```diff
@@ -107,7 +107,7 @@ jobs:
         run: sudo xcode-select -switch /Applications/Xcode_13.3.app
 
       - name: Build MacOS Client
-        run: npm run action cordova/build macos
+        run: npm run action cordova/build macos -- --verbose
```
Comment (bemasc) [inline .github/workflows/build_and_test_debug.yml:110]: Wait, does this work or not?

**Review by bemasc — APPROVED** [review]

**Pull request merged**